src/site/README.md

```diff
@@ -0,0 +1,309 @@
+
+The simplest way of getting your page's `entityBundle` is by leveraging the usual avenues on Slack to connect with the CMS team (primarily the `#cms-support` channel) or by simply asking your supporting CMS engineer what the value is. This value can be considered the "hand-off" to the front-end team so that templating can begin.
+
+</details>
+
+<details>
+
+<summary>2. Write the GraphQL query</summary>
+
+The CMS provides a tool called the [GraphQL Explorer](https://prod.cms.va.gov/graphql/explorer) to help write and debug GraphQL queries. You may be familiar with this tool from other GraphQL servers. You can leverage the value of the `entityBundle` determined in the former step to get started writing your query. If you are not familiar with GraphQL queries, the GraphQL website provides fantastic [documentation](https://graphql.org/learn/queries/) we recommend reading.
+
+A GraphQL query for retrieving pages will consist of a `nodeQuery` with a `filter` applied to narrow down results to only nodes belonging to a certain `entityBundle`. Consider this GraphQL snippet used for getting all pages of type `landing_page`. In this example, `landing_page` is the value of the `entityBundle`.
+
+```graphql
+{
+  nodeQuery(limit: 1, filter: {conditions: [{field: "type", value: ["landing_page"]}]}) {
+    entities {
+      ... on NodeLandingPage {
+        entityId
+      	entityBundle
+        title
+        fieldIntroText
+      }
+    }
+  }
+}
+```
+
+The response JSON for this GraphQL query will contain a single instance of the page-type `landing_page`. Here is the response JSON as of writing.
+
+```json
+{
+  "data": {
+    "nodeQuery": {
+      "entities": [
+        {
+          "entityId": "79",
+          "entityBundle": "landing_page",
+          "title": "VA records",
+          "fieldIntroText": "Access your VA records and documents online to more easily manage your benefits."
+        }
+      ]
+    }
+  }
+}
+```
+
+Your complete query will be restructured so that your JS module exports a GraphQL fragment for use in the preview server as well as as a standalone `nodeQuery` for use in the content build. It should also contain an additional `filter` - a boolean field called `status` that is used to toggle only draft vs. published content. Here is a complete example of what your module may look like. _Note - Hopefully, soon the preview server will be updated to use the same query as the content build. This doc wil be updated once that happens._
+
+```js
+const examplePageFragment = `
+  fragment examplePageFragment on NodeLandingPage {
+    entityId
+    entityBundle
+    title
+    fieldIntroText
+  }
+`;
+
+const GetExamplePages = `
+  ${examplePageFragment}
```

Comment:
Where does `examplePageFragment` come from? Does this need to correspond to something in the CMS?

Reply:
It doesn't mean anything to the CMS. It's a totally arbitrary. Clients can name their fragments whatever they want but in practice they should obviously be more descriptive than `examplePageFragment`.

Reply:
Might be a good idea to add some sort of explanation of the graphql query file structure. It looks like we have folders for specific fragment types (blocks, facilities, files, navigation, etc.).

Do all "top-level" page queries need to live in `src/site/stages/build/drupal/graphql` (root level)?